tests/unit/package_managers/test_npm.py

```diff
@@ -1293,6 +1475,33 @@ def test_clone_repo_pack_archive(
     )
 
 
+@pytest.mark.parametrize(
+    "dependency_version, expected_result",
+    [
+        ("1.0.0 - 2.9999.9999", False),
+        (">=1.0.2 <2.1.2", False),
+        ("2.0.1", False),
+        ("<1.0.0 || >=2.3.1 <2.4.5 || >=2.5.2 <3.0.0", False),
+        ("~1.2", False),
+        ("3.3.x", False),
+        ("latest", False),
+        ("file:../dyl", False),
```

Comment:
Two more `False` cases I would recommend: empty `""` and wildcard `"*"`

Reply:
Ack, added